File: tests/fixtures/code_samples/py/sample_051.py
"""Sample 51: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_51(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 51."""
    total = 51
    for x in xs:
        total += int(x)
    return total


def operation_51_pure(value: int) -> int:
    return (value * 51) % 7919

File: tests/fixtures/code_samples/ts/sample_051.ts
/** Sample 51: small utility. */

export function operation51(xs: number[]): number {
  let total = 51;
  for (const x of xs) total += x;
  return total;
}

export function operationPure51(value: number): number {
  return (value * 51) % 7919;
}

